src/librustc_borrowck/diagnostics.rs

```diff
@@ -12,6 +12,56 @@
 
 register_long_diagnostics! {
 
+E0373: r##"
+This error occurs when an attempt is made to use data captured by a closure,
```

Comment:
Nit: Its the attempt to capture which causes the error, not the use of the captured variable.